tests/stdlib/tstrscans.nim

```diff
@@ -0,0 +1,82 @@
+import strscans
```

Comment:
please add a discard section with an empty output string here to ensure the test is really run.